EIPS/eip-3372.md

```diff
@@ -0,0 +1,87 @@
+---
+eip: 3372
+title: Apply minor modifications to the Ethash algorithm to break current ASIC implementations
```

Comment:
```suggestion
title: 5 FNV primes for ethash
```
Title should be incredibly short and focus on the technical change, not the motivation for it.